contracts/MixinResolver.sol

```diff
@@ -50,25 +75,6 @@ contract MixinResolver is Owned {
         return _foundAddress;
     }
 
-    // Note: this could be made external in a utility contract if addressCache was made public
-    // (used for deployment)
-    function isResolverCached(AddressResolver _resolver) external view returns (bool) {
```

Comment:
Looks like this function was not used by any of the contracts. Besides, the new public isCached variable provides the same functionality. So looks ok to remove :+1: